publications.markdown

```diff
@@ -52,7 +52,7 @@ If you want to change the style of the table, please look at the publication tag
 <td>
     <b>GraphDistNet: A Graph-based Collision-distance Estimator for Gradient-based Trajectory</b><br>
     Yeseung Kim, Jinwoo Kim, Daehyung Park<br>
-    <i>IEEE Robotics and Automation Letters (RA-L), Oral presentation in IEEE IROS, 2022</i> <br>
+    <i>IEEE Robotics and Automation Letters (RA-L), Oral presentation in IEEE International Conference on Intelligent Robots and Systems (IROS), 2022</i> <br>
```

Comment:
Please, remove ", Oral presentation in IEEE International Conference on Intelligent Robots and Systems (IROS)"

Reply:
removed in [31e1bc4](https://github.com/rirolab/rirolab.github.io/pull/7/commits/31e1bc4dc078fa94160c104832ee220330553084)